blockchain/blockchain.go

```diff
@@ -409,7 +409,24 @@ func (bc *blockchain) Nonce(addr string) (uint64, error) {
 
 // CreateState adds a new account with initial balance to the factory
 func (bc *blockchain) CreateState(addr string, init *big.Int) (*state.Account, error) {
```

Comment:
I realize this method is only used for testing now?

Reply:
so far yes
i will refactor it in next PR, given our discussion about Load/CreateState attack